data/content.json

```diff
@@ -77,7 +77,7 @@
 		},
 		{
 			"name": "Accordion",
-			"version": "2.1.0",
+			"version": "2.1.1",
```

Comment:
Does this file know more than we do? 🤔  https://github.com/mendix/widgets-resources/blob/master/packages/pluggableWidgets/accordion-web/package.json#L4 (I don't see a patch)

Reply:
it's the native one

Reply:
Oopsies, my bad